From f1b83d78e1920b5ec956985880c5ffbe8e88923b Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Thu, 13 May 2010 19:28:07 -0400
Subject: [PATCH 322/391] lttng-trace-format-bump-minor-to-2.4

lttng trace format bump minor to 2.4

Bump minor number to 2.4 to replace reverted alignment modification.
Will be followed by a bump to 2.6 later in the patchset (after a bump to 2.5).

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 include/linux/ltt-tracer.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/linux/ltt-tracer.h b/include/linux/ltt-tracer.h
index ea25ad3..2b19b4c 100644
--- a/include/linux/ltt-tracer.h
+++ b/include/linux/ltt-tracer.h
@@ -534,7 +534,7 @@ size_t ltt_read_event_header(struct ltt_chanbuf_alloc *bufa, long buf_offset,
 #define LTT_DEFAULT_N_SUBBUFS_HIGH	2
 #define LTT_TRACER_MAGIC_NUMBER		0x00D6B7ED
 #define LTT_TRACER_VERSION_MAJOR	2
-#define LTT_TRACER_VERSION_MINOR	3
+#define LTT_TRACER_VERSION_MINOR	4
 
 /**
  * ltt_write_trace_header - Write trace header
-- 
1.6.5.2

